docs/en/concepts-and-designs/manual-sdk.md

```diff
@@ -1,9 +1,12 @@
 # Manual instrument SDK
 Our incredible community has contributed to the manual instrument SDK.
 - [Rust](https://github.com/apache/skywalking-rust). Rust SDK follows the SkyWalking format.
-- [Go2Sky](https://github.com/SkyAPM/go2sky). Go SDK follows the SkyWalking format.
+- [Golang](https://github.com/apache/skywalking-go). Go SDK follows the SkyWalking format.
```

Comment:
skywalking-go is an auto-instrumentation, you could consider to removing this from here.

Reply:
So, an auto-instrumentation is not a SDK because users don't need to write any codes. Am I right?

How about the archived one? What is your suggestion, keep or remove?

Reply:
> So, an auto-instrumentation is not a SDK because users don't need to write any codes. Am I right?

Yes. Some of the agents(auto-instrumentation) also provide SDKs through specific ways, such as Java agent toolkit or WIP go agent toolkit. But generally, yes.

> How about the archived one? What is your suggestion, keep or remove?

You could keep that. The protocol is there, in theory, we still are able to receive data from that agent. We just don't maintain or test it anymore.